grid.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+import Ogre
+
+class GridFloor:
+    def __init__(self, scene, material_name):
+        self.scn_mgr = scene
+        self.material_name = material_name
+        self.drawQueue = 1
```

Comment:
why does this need its own renderQueue and alpha blending? Wouldnt setting the grid color to a dark grey do the same?

Reply:
I think it does not really need renderQueue and alpha blending.

Most likely those were features from the place I took the code: https://wiki.ogre3d.org/Peter%27s+Grid+System

The thing I *do* need is vertex colors.

Reply:
you could create such a material in a few lines of code like:
https://github.com/OGRECave/ogre/blob/fb8193bd5e5dd43e2d72b67fd0f9de2127cc8bcb/OgreMain/src/OgreDefaultDebugDrawer.cpp#L31-L34

Reply:
Thanks!